manual/manual/tutorials/coreexamples.etex

```diff
@@ -677,6 +677,36 @@ print_expr e; print_newline ();;
 print_expr (deriv e "x"); print_newline ();;
 \end{caml_example}
 
+There is a "printf" function in "Printf" module that allows you to make
+formatted output more concisely. It models the behavior of the "printf"
+function from the C standard library.
+The "printf" function takes a format string that describes the output values and
+their format, and arguments for output that correspond to the formatting
+directives in the format string.
+\begin{caml_eval}
+open Printf;;
+\end{caml_eval}
+\begin{caml_example}{toplevel}
+printf "%i + %i is an integer value, %F * %F is a float, \"%s\"\n"
+3 2 4.5 1. "this is a string";;
+\end{caml_example}
+The printf function in the OCaml language performs type checking of arguments.
```

Comment:
I think it would made sense to rephrase this sentence to  `OCaml type system checks that the type of the arguments and the specifiers are compatible`, to avoid the idea that `printf` is doing any type checking on its own.